lbjava-examples/pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
```

Comment:
Sorry for being nitpicky. Would be great if you add a readme file for `lbjava-examples/src/main/java/edu/illinois/cs/cogcomp/lbjava/examples/neuralnet/`, explaining what is input/output, where to get the data, and what to expect after running the system.